content/docs/index.md

```diff
@@ -9,6 +9,15 @@ last_edited: '2020-11-26T14:37:54.481Z'
 - The **TinaCMS toolkit**, a suite of NPM packages that enable developers to quickly create a custom content management frontend for a website.
 - **Tina Cloud**, a hosted service designed to make Git-based content management more powerful and accessible for cross-functional teams.
 
+<div class="callout">
+<img className="learnImage" src="../img/tina-laptop.png" alt="" />
+  <div>
+  <h3>Ready to get started?</h3>
+  <p>Read our getting started guide to get set up in no time.</P>
```

Comment:
```suggestion
  <p>Get a Tina running website in no time!</p>
```

Reply:
@DirtyF  How about "Get a website running with Tina Cloud in no time!"?

Reply:
even better!

Reply:
Done.